src/wcl/tracing.h

```diff
@@ -78,14 +81,15 @@ class FormatSubscriber : public Subscriber {
   ~FormatSubscriber() override{};
 };
 
-class FatalEventSubscriber : public Subscriber {
+class FilteredStreamSubscriber : public Subscriber {
```

Comment:
The format for the user is intentionally different that the format for the log. If we are doing forwarding then we can't change the output format

FormatSubscriber Format: `[tag = value, tag = value, tag = value] message`
FilteredStreamSubscriber Format: `[level] message`

Reply:
That's a second orthogonal issue as I see it. The FilteredStreamSubscriber as is, does multiple highly specific things and it can be decomposed into much simpler parts. If FilteredStreamSubscriber were defined in the wake tool I think this would be fine but its currently defined in wcl where it seems far too specific.

Reply:
BTW, maybe a resolution you'd be happier with is just to move FilteredStreamSubscriber to tools/wake/main.cpp or tools/wake/filter_subscriber.h or something like that?

Reply:
Yeah, I'm more than happy to move where it lives